lib/nats.go

```diff
@@ -7,4 +7,5 @@ const (
 	NatsMarketOrdersDeduped = "marketorders.deduped"
 	NatsMapDataIngest       = "mapdata.ingest"
 	NatsMapDataDeduped      = "mapdata.deduped"
+	NatsSkillDataIngest     = "skills"
```

Comment:
Can you please adjust this block of consts to look more like this:

```go
// Public Topics
topic = "value"

// Private Topics
topic = "value"
```

Just to help keep it organized. :)